tests/command.functions.test.js

```diff
@@ -560,6 +560,39 @@ test('should trigger background function from event', async (t) => {
     })
   })
 })
+;['esbuild', undefined].forEach((nodeBundler) => {
```

Comment:
We use [this pattern](https://github.com/netlify/cli/blob/efc0b48db300f0338482290b9a72bc9eb99504a1/tests/serving-functions.test.js#L12) in a different file to test against multiple bundlers. Do you think we could do the same here for consistency?

Reply:
adapted in 3fa4c14.